idl/partition_checksum.idl.hh

```diff
@@ -154,3 +154,13 @@ struct repair_update_system_table_request {
 
 struct repair_update_system_table_response {
 };
+
+struct repair_flush_hints_batchlog_request {
+    utils::UUID repair_uuid;
+    std::list<gms::inet_address> target_nodes;
+    uint32_t hints_timeout_in_seconds;
+    uint32_t batchlog_timeout_in_seconds;
```

Comment:
Maybe we can pass std::chrono::seconds?

Reply:
I thought about in the first but I chose to use uint32 because it is smaller (sizeof(std::chrono::seconds) = 8) and I think uint32  is more than enough.

Reply:
Um, this message is sent a few times per day. We should aim for reducing the chance of misuse.

Reply:
OK. Done.